cmd/destroy.go

```diff
@@ -209,7 +210,7 @@ func destroyLab(ctx context.Context, c *clab.CLab) (err error) {
 	}
 
 	// delete lab management network
-	log.Infof("Deleting network '%s'...", c.Config.Mgmt.Network)
```

Comment:
why do you remove the explicit message with a network name?

Reply:
because that message is  runtime dependent ... in containerd the bridge name is stored in c.Config.Mgmt.Bridge...
hence this message was replaced with the next line `log.Infof("Cleanup networking")` and the final message what that cleanup means comes from the runtime.

Reply:
the implementation is runtime specific, but as far as clab is concerned, there is always a management network with a certain name.

So what I'd propose is the following

1. do the check if the deletion of the network is needed (not bridge and not keepMgmtNet) here, and do the log.Info if the deletion is needed
2. Output the original `log.Infof("Deleting network '%s'...", c.Config.Mgmt.Network)` then, since now it will mean that the we will make an attempt to delete the network
3. the DeleteNet runtime function will not emit any Info messages

Reply:
Take a look at the actual approach ...
The issue again is, that we need to delegate the decision to the runtime if something can be deleted or not.
Names are for Docker defined in the c.Config.Mgmt.Network and for containerd in c.Config.Mgmt.Bridge because we do not need and additional Network abstraction here ... 
Also take a look at the comments on the struct
https://github.com/srl-labs/containerlab/blob/eb969dba156c9f7063cfe9f7fa797104ec7dc110/types/types.go#L38-L44